GetDKAN/dkan2
# Better integration between dkan_api, dkan_harvest, dkan_schema, and interra_api

**QA Steps**

1. Enable dkan_dummy_content
2. Make sure you have Drush 9 installed
3. Run `drush dkan-dummy-content:create`
4. The site should have some content
5. Run `drush dkan-dummy-content:remove`
6. All of the content should be gone
